src/Stratis.Bitcoin/Base/ChainHeadersBehavior.cs

```diff
@@ -286,7 +292,7 @@ private void AttachedPeer_MessageReceived(NetworkPeer peer, IncomingMessage mess
                         if (pendingTipBefore != this.pendingTip)
                             this.logger.LogTrace("Pending tip changed to '{0}'.", this.pendingTip);
 
-                        if (this.pendingTip.ChainWork > this.Chain.Tip.ChainWork)
+                        if (this.pendingTip != null && this.pendingTip.ChainWork > this.Chain.Tip.ChainWork)
```

Comment:
() pls

Reply:
but pending tip can't be null here

Reply:
why? I think it can. 
Since we no longer set pending tip to be equal to our tip at the very start it is possible that pending tip could be null here.

Reply:
OK I see, it is the new thing. So () pls here.

Reply:
Already added before.